lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -13,6 +13,8 @@
 
 LOG_MODULE_REGISTER(multicell_location_nrf_cloud, CONFIG_MULTICELL_LOCATION_LOG_LEVEL);
 
+static char jwt_buf[1024];
```

Comment:
nrf_cloud.h says that the buffer "recommended size >= 600 bytes", so this could probably be smaller.

Reply:
I made it 600 and let's hope it works. I tried and 400 wasn't enough and 500 worked for me. I don't know how much the size can vary.